### ISSUE TITLE
Placement des pictogrammes

### ISSUE THREAD

**Issue by Vincent-Espagne-1** [1twitif/social-viz#4]

**Comment by Vincent-Espagne-1** [comment]
Sur les cartes Argentine et Total

**Comment by GammaNu** [comment]
Il s'agit bien de les utiliser pour représenter les acteurs associé ? (je pondère en partant de ce principe)

S'il s'agit d'un placement spatial, j'ai besoin de plus d'info sur ce qui est souhaité.

**Comment by Vincent-Espagne-1** [comment]
Oui, à chaque acteur, un picto en fonction de son type et de son activité.
Le placement spatial devrait se faire automatiquement, les acteurs d'un même type regroupés dans la même zone.

**Comment by Vincent-Espagne-1** [comment]
3 (nécessaire pour réussir la démonstration)
